Block/Adminhtml/Reindex/Edit/Form.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Module Algolia Algoliasearch
+ */
+namespace Algolia\AlgoliaSearch\Block\Adminhtml\Reindex\Edit;
+
+use \Magento\Backend\Block\Widget\Form\Generic;
+
+/**
+ * Class: Form
+ */
```

Comment:
Can you remove this comment?

Reply:
It's redundant as it doesn't say anything more than is on the line below - `class Form` :)